src/site/content/en/blog/mediastreamtrack-insertable-media-processing/index.md

```diff
@@ -0,0 +1,252 @@
+a given speaker, like left or right in a stereo audio track.
+
+## What is insertable streams for `MediaStreamTrack`?
+
+The core idea for insertable streams for `MediaStreamTrack` is to expose the content of a
+`MediaStreamTrack` as a collection of [streams](/streams/) (as defined by the WHATWG
+[Streams API](https://streams.spec.whatwg.org/)), which can be manipulated to introduce new
+components.
+
+## Use cases
+
+Use cases for insertable streams for `MediaStreamTrack` include, but are not limited to:
+
+- Video conferencing gadgets like "funny hats" or virtual backgrounds.
+- Voice processing like software [vocoders](https://en.wikipedia.org/wiki/Vocoder).
+
+## Current status {: #status }
+
+<div class="w-table-wrapper">
+
+| Step                                     | Status                   |
+| ---------------------------------------- | ------------------------ |
+| 1. Create explainer                      | [Complete][explainer]    |
+| 2. Create initial draft of specification | [In Progress][spec]      |
+| 3. Gather feedback & iterate on design   | [In progress](#feedback) |
+| 4. Origin trial                          | Not started              |
+| 5. Launch                                | Not started              |
+
+</div>
+
+## How to use insertable streams for `MediaStreamTrack`
+
+### Enabling via chrome://flags
+
+To experiment with insertable streams for `MediaStreamTrack` locally, without an origin trial token,
+enable the `#enable-experimental-web-platform-features` flag in `chrome://flags`.
+
+### Feature detection
+
+You can feature-detect insertable streams for `MediaStreamTrack` support as follows.
+
+```js
+if ('MediaStreamTrackProcessor' in window && 'MediaStreamTrackGenerator' in window) {
+  // Insertable streams for `MediaStreamTrack` is supported.
+}
+```
+
+### Core concepts
+
+Insertable streams for `MediaStreamTrack` objects conceptually splits the `MediaStreamTrack` into
+two components:
+
+- The `TrackProcessor`, which consumes a `MediaStreamTrack` object's source and generates a stream
+  of media frames ([`VideoFrame`](https://w3c.github.io/webcodecs/#videoframe-interface) or
+  [`AudioFrame`](https://w3c.github.io/webcodecs/#audioframe-interface)). You can think of this as a
+  track sink that is capable of exposing the unencoded frames from the track to a `ReadableStream`
+  and that exposes a control channel for signals going in the opposite direction.
+- The `TrackGenerator`, which consumes a stream of media frames and exposes a `MediaStreamTrack`
+  interface, so that it can be used anywhere a `MediaStreamTrack` is currently attached. It takes
+  video frames as input, and emits control signals that result from subsequent processing.
+
+#### The `MediaStreamTrackProcessor`
+
+A `MediaStreamTrackProcessor` object exposes two properties:
+
+- `readable`: Allows reading the frames flowing through the `MediaStreamTrack`. If the track is a
```

Comment:
The `readable` property sounds like it should be a boolean, but it's not?

Reply:
It’s a `readable` stream, as in web.dev/streams/, not meant as `isReadable`.